### PR TITLE
#171204743 setup multilingual access

### PR DESCRIPTION
**Pull request by hakoemmy**
#### What does this PR do?
- This PR adds French and English languages
#### Description of Task to be completed?
 - Install ``i18n-2`` module
 - Add JSON file for English 
-  Add JSON file for french
- Allow user to choose his/her preferred language
- Make English default language
#### How should this be manually tested?
- Clone the [repo](https://github.com/andela/octopus-bn-backend.git) and 
- Run ``` npm install ``` in terminal
- Run ``` npm run dev ``` to start localhost server
- Launch your postman and send a request to ``` localhost:3000/api/v1/?lang=fr ```
- You can change ``lang`` query string to ``en`` to translate to English accordingly
- Keep in mind that, we have two languages[``en , fr``] supported at the moment.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171204743](https://www.pivotaltracker.com/story/show/171204743)
### Screenshots (if appropriate)
Screen Shot 2020-02-12 at 12 53 45
Screen Shot 2020-02-12 at 12 58 56
Screen Shot 2020-02-12 at 13 00 29

#### Questions:
N/A

### CODE REVIEW

File: locales/fr.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "Welcome":"Bienvenue à Barefoot Nomad",
+    "NotFound":"Pas trouvé",
+    "AppDesc":"Rendre les voyages et l'hébergement en entreprise faciles et pratiques"
+}
```
Comment (AbdoulNuru) [inline locales/fr.js]: A new line is needed at the of the file

**Review by AbdoulNuru — CHANGES_REQUESTED** [review]

File: locales/en.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "Welcome":"Welcome to Barefoot Nomad",
+    "NotFound":"Not Found",
+    "AppDesc":"Making company travel and accomodation easy and convinient"
+
+}
```
Comment (AbdoulNuru) [inline locales/en.json]: Also here the new line is required

**Review by AbdoulNuru — CHANGES_REQUESTED** [review]

File: locales/fr.js (hunk 1 above)
Comment (hakoemmy) [reply locales/fr.js]: Thank you @AbdoulNuru . Allow me to fix those ESlint issues.

File: locales/en.json (hunk 2 above)
Comment (hakoemmy) [reply locales/en.json]: @AbdoulNuru  thank you. I've fixed this

**Review by AbdoulNuru — APPROVED** [review]

**Review by izzett222 — APPROVED** [review]

**Review by RusimbiPatrick — APPROVED** [review]

**Comment by RusimbiPatrick** [comment]
Good job!!

**Comment by hakoemmy** [comment]
> Good job!!

It's my pleasure to hear that

**Review by kcharles52 — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Pull request merged**